features/org.wso2.developerstudio.eclipse.bps.feature/feature.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Missing License Header. Check this for all new and existing files. If you did a modification to existing file, update current year in license text.